layouts/partials/footer.hbs

```diff
@@ -5,7 +5,19 @@
   <div class="container">
     <div class="openjsfoundation-footer">
       <p>Copyright <a href="https://openjsf.org">OpenJS Foundation</a> and Node.js contributors. All rights reserved. The <a href="https://openjsf.org">OpenJS Foundation</a> has registered trademarks and uses trademarks.  For a list of trademarks of the <a href="https://openjsf.org">OpenJS Foundation</a>, please see our <a href="https://trademark-policy.openjsf.org">Trademark Policy</a> and <a href="https://trademark-list.openjsf.org">Trademark List</a>.  Trademarks and logos not indicated on the <a href="https://trademark-list.openjsf.org">list of OpenJS Foundation trademarks</a> are trademarks&trade; or registered&reg; trademarks of their respective holders. Use of them does not imply any affiliation with or endorsement by them.</p>
-      <p><a href="https://openjsf.org">The OpenJS Foundation</a> | <a href="https://terms-of-use.openjsf.org">Terms of Use</a> | <a href="https://privacy-policy.openjsf.org">Privacy Policy</a> | <a href="https://bylaws.openjsf.org">Bylaws</a> | <a href="https://code-of-conduct.openjsf.org">Code of Conduct</a> | <a href="https://trademark-policy.openjsf.org">Trademark Policy</a> | <a href="https://trademark-list.openjsf.org">Trademark List</a> | <a href="https://www.linuxfoundation.org/cookies">Cookie Policy</a></p>
+      <p style="display:inline">
+        <a href="https://openjsf.org">The OpenJS Foundation</a> | 
+        <a href="https://terms-of-use.openjsf.org">Terms of Use</a> | 
+        <a href="https://privacy-policy.openjsf.org">Privacy Policy</a> | 
+        <a href="https://bylaws.openjsf.org">Bylaws</a> | 
+        <a href="https://code-of-conduct.openjsf.org">Code of Conduct</a> | 
+        <a href="https://trademark-policy.openjsf.org">Trademark Policy</a> | 
+        <a href="https://trademark-list.openjsf.org">Trademark List</a> | 
+        <a href="https://www.linuxfoundation.org/cookies">Cookie Policy</a>
+        </p>
+        <div style="display:inline;font-size: 15px;">
```

Comment:
We should avoid using inline styles as it's hard to maintain.

Reply:
@osk2 : OK it's fixed.